pkg/controller/backup_controller.go

```diff
@@ -532,6 +543,28 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 		fatalErrs = append(fatalErrs, err)
 	}
 
+	// Empty slices here so that they can be passed in to the persistBackup call later, regardless of whether or not CSI's enabled.
+	// This way, we only make the Lister call if the feature flag's on.
+	var volumeSnapshots []*snapshotv1beta1api.VolumeSnapshot
+	var volumeSnapshotContents []*snapshotv1beta1api.VolumeSnapshotContent
+	if features.IsEnabled("EnableCSI") {
+		selector := labels.SelectorFromSet(map[string]string{velerov1api.BackupNameLabel: backup.Name})
+
+		// TODO(nrb-csi): Only run listers methods if the listers aren't nil, just in case
+		volumeSnapshots, err = c.volumeSnapshotLister.List(selector)
+		if err != nil {
+			//TODO: Is this right?
```

Comment:
seems right, this will output it to both the backup log file and to stdout

Reply:
The reason I had the question was whether or not it should be considered an error. If we're backing up with CSI and volumes included in the backup is >= 1, then we should have no error. If volumes included in the backup == 0, then I expect a 404; I'll need to test this case.
 
As written, I think this code will throw an error for backups that don't include volumes, so we probably want to add that condition.